metricbeat/docs/fields.asciidoc

```diff
@@ -6040,6 +6040,194 @@ Used inodes
 
 
 [float]
+== events Fields
+
+The Kubernetes events metricset collects events that are generated by objects running inside of Kubernetes
+
+
+
+[float]
+=== kubernetes.events.count
+
+type: long
+
+Count field records the number of times the particular event has occured
+
+
+[float]
+=== kubernetes.events.firstOccurrenceTimestamp
```

Comment:
I would suggest to change this to `*.event.timestamp.first_occurence`and below to `*.event.timestamp.last_occurence`.